ext/droption/droption.h

```diff
@@ -371,6 +371,8 @@ class droption_parser_t {
     convert_from_string(const std::string s1, const std::string s2) = 0;
     virtual bool
     clamp_value() = 0;
+    virtual bool
+    is_negative_input(const std::string &s) = 0;
```

Comment:
Does this routine need to be virtual? We can probably make it static, right?

Reply:
Done.